SingularityUI/app/assets/index.mustache

```diff
@@ -47,10 +47,10 @@
         };
     </script>
     <script src="{{{staticRoot}}}/js/vendor.bundle.js"></script>
-    <script src="{{{staticRoot}}}/js/app.js"></script>
     {{#extraScript}}
     {{{extraScript}}}
     {{/extraScript}}
+    <script src="{{{staticRoot}}}/js/app.js"></script>
```

Comment:
I don't think so (wolf can verify). Sentry does need to be loaded before our code though.